Server/Services/openMeteoService.js

```diff
@@ -0,0 +1,38 @@
+const googleService = require('./googleService');
+const http = require('http');
+
+module.exports = {
+    WeatherRainingOrNot: function(res) {
+        var request = http.get('http://api.open-meteo.com/v1/forecast?latitude=47.22&longitude=-1.55&hourly=rain', function (response) {
+            var buffer = ""
+            var data;
+
+            response.on("data", function (chunk) {
+                buffer += chunk;
+            }); 
+            response.on("end", function (err) {
+                data = JSON.parse(buffer);
+                // get date and time info
+                var date = new Date().toISOString().slice(0, 10);
+                var hour = new Date().getHours();
+                console.log(date, 'à', hour, 'heures')
+                // Search for data corresponding to the current date
+                for (var i = 0; i < data.hourly.time.length; i++) {
+                    // Compare date and time
+                    if (data.hourly.time[i].slice(0, 10) == date && data.hourly.time[i].slice(11, 13) == hour) {
+                        // Check if it is raining
+                        if (data.hourly.rain[i + 1] > 0) {
```

Comment:
I don't know if you can make it easily but it would be great if we could pass an argument (like an enum of the different meteos (raining/shining/windy/...) to get the type of meteo we want